release.nix

```diff
@@ -19,7 +19,7 @@ let
     export PATH=${x.pkgs.hevm}/bin:${x.pkgs.jq}/bin:$PATH
     ${x.pkgs.hevm}/bin/hevm compliance \
       --tests ${ethereum-test-suite x} \
-      --skip "(Create2Recursive|Create1000|recursiveCreateReturn|underflowTest|walletRemoveOwnerRemovePending|Return5000|randomStatetest177|loopExp|loopMul)" \
+      --skip "(Create2Recursive|Create1000|recursiveCreateReturn|underflowTest|walletRemoveOwnerRemovePending|Return5000|randomStatetest177|loopExp|loopMul|FirstByte)" \
```

Comment:
why are we skipping `FirstByte`?

Reply:
Those tests do not have a `poststate`, just the poststate hash, which we are not deciphering at the moment

Reply:
can we document that as a comment in this file? Would be awesome to have reasons for all the skips here actually, but that doesn't have to be done now.